README.md

```diff
@@ -4,16 +4,18 @@ This repository contains **MATLAB** code samples for **Zivid**.
 
 ## Samples list
 
-- [**ReadZDF**](https://github.com/zivid/matlab-samples/blob/master/ReadZDF.m) - Import ZDF point cloud.
-- [**ZDF2PLY**](https://github.com/zivid/matlab-samples/blob/master/ZDF2PLY.m) - Convert ZDF point cloud to PLY file format.
-- [**ZDF2CSV**](https://github.com/zivid/matlab-samples/blob/master/ZDF2CSV.m) - Convert ZDF point cloud to CSV format.
-- [**ZDF2TXT**](https://github.com/zivid/matlab-samples/blob/master/ZDF2TXT.m) - Convert ZDF point cloud to TXT format.
-- [**DownsamplePointCloud**](https://github.com/zivid/matlab-samples/blob/master/DownsamplePointCloud.m) - Import ZDF point cloud and downsample it.
-- [**zdfread**](https://github.com/zivid/matlab-samples/blob/master/zdfread.m) - Function for reading a .ZDF file.
-- [**zdf2image**](https://github.com/zivid/matlab-samples/blob/master/zdf2image.m) - Function for exporting a 2D color image from from a .ZDF file.
-- [**downsample**](https://github.com/zivid/matlab-samples/blob/master/downsample.m) - Function for downsampling a Zivid point cloud.
+There are two main categories of samples: **Camera** and **Applications**. The samples in the **Camera** category focus only on how to use the camera. The samples in the **Applications** category use the output generated by the camera, such as the 3D point cloud, a 2D image or other data from the camera. These samples shows how the data from the camera can be used.
 
-Note: The functions are independent of any MATLAB toolboxes, while sample scripts require [**Image Processing Toolbox**](https://se.mathworks.com/products/image.html) and [**Computer Vision Toolbox**](https://se.mathworks.com/products/computer-vision.html).
+- **Applications**
+  - **Basic**
+    - **FileFormats**
+      - [**zdfread**][zdfread-url] - This examples shows how to read a Zivid point cloud from a .zdf file.
+      - [**ReadIterateZDF**][ReadIterateZDF-url] - This example shows how to import and display a Zivid point cloud from a .zdf file.
```

Comment:
Maybe change description to:
"This example shows how to import and display a Zivid point cloud. In addition it shows how to iterate through, and extract individual points. Note that this example depends on [**zdfread**][zdfread-url] to read .zdf files."

Reply:
I like the first part. I will for now skip 'Note that this example depends on [zdfread][zdfread-url] to read .zdf files.' If we were to put that, we would then need to put the same sentence for other samples that depend on zdfread (Downsample and convertzdf), and that would be verbose.

Reply:
Done